site/src/GitHubStars/GitHubStars.tsx

```diff
@@ -25,11 +25,14 @@ export const GitHubStars = () => {
       );
       const { stargazers_count: count } = await res.json();
 
-      setStars(
-        Math.abs(count) > 999
-          ? `${(Math.sign(count) * (Math.abs(count) / 1000)).toFixed(1)}k`
-          : `${Math.sign(count) * Math.abs(count)}`,
-      );
+      let starCount = null;
+      if (typeof count === 'number') {
+        starCount =
+          Math.abs(count) > 999
+            ? `${(Math.sign(count) * (Math.abs(count) / 1000)).toFixed(1)}k`
+            : `${Math.sign(count) * Math.abs(count)}`;
+      }
+      setStars(starCount);
```

Comment:
Should we just call `setStars` conditionally? No reason to call `setStars(null)` yeah?

Reply:
Just realized we only call this the once on mount 🤦‍♂️, yep I'll update